MIGRATION-GUIDE.md

```diff
@@ -0,0 +1,40 @@
+
+# Migrating to Redux SDK v2.0.0
+
+Redux SDK v2.0.0 introduces a breaking change that you should consider when migrating from a previous version.
+
+If you were passing the `core.trafficType` option to the SDK configuration object, you should remove it since it is no longer supported.
+The `trafficType` must be passed as an argument of the `track` helper function. For example:
+
+```js
+import { initSplitSdk, track } from '@splitsoftware/splitio-redux'
+
+const CONFIG = {
+  core: {
+    authorizationKey: YOUR_CLIENT_SIDE_SDK_KEY,
+    key: USER_KEY,
+    trafficType: 'user'
```

Comment:
I didn't think of this before: 
Here the `trafficType` setting is optional right (defaults to 'user')? But now the `track()` `trafficType` parameter is required? If so, we should explain what they should do if they are not be setting the `core.trafficType` setting in their v1.x code.

Reply:
In v1 of Redux SDK (and v10 of JS SDK), you were able to choose were to pass the `trafficType`: either in the config or the `track` method call. But if you passed in both places, you got an error when calling the `track` method. 

Now, to avoid confusion, the `trafficType` must always be passed to the `track` method. It is not optional, but maybe, in the future it might be optional (with 'user' as default maybe), but not at this moment.

Reply:
So, if they are not setting the `core.trafficType` setting in their v1.x code, they should not change anything: they must be already sending the `trafficType` with the `track` call.

Reply:
This makes perfect sense! Thanks, Emi!